Updates to EMA Cluster Measure

bijujoseph opened the pull request:
https://jira.cms.gov/browse/QPPQS-443
Summary of changes
```
1)	Registry
a)	Measure 110 is removed from OsteoporisCare
b)	Measure 226 is removed from HeartFailureCare
2)	Claims
a)	Measure 117 is ADDED to EyeCare
b)	NEW Cluster ColonscopyCare
i)	Measure 130 is ADDED to NEW ColonscopyCare cluster
ii)	Measure 425  is ADDED to NEW ColonscopyCare cluster
c)	NEW Cluster EndoscopyandPolyp Surveillance
i)	Measure 185 is ADDED to EndoscopyandPolyp Surveillance
ii)	Measure 320 is ADDED to EndoscopyandPolyp Surveillance
```

kencheeto commented on .gitignore:
```diff
@@ -7,3 +7,4 @@ ecqm_eligibleclinician_jan2017
 Claims-Registry-Measures.zip
 Claims-Registry-Measures
 Web-Interface-Measures.zip
+.idea/
```
might be easier to add this to a global gitignore, if you've got one

bijujoseph replied:
sounds good

bijujoseph commented on package-lock.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "0.0.40",
+  "version": "1.0.0-alpha.1",
```
sounds good, will take care

bijujoseph commented on test/clinical-clusters/clinical-clusters-spec.js:
```diff
@@ -36,4 +36,74 @@ describe('benchmarks getter functions', () => {
+      var osteoporosisCare = theCluster.clinicalClusters.find(c => c.name === 'heartFailureCare');
+      assert.notInclude(osteoporosisCare.measureIds, '226');
+    });
+  });
```
Yes! just asserting that the cluster generator rightly processes the raw data. I got your point, this can go too lengthy overtime. Like every change of an attribute in a measure can break some or the other thing in downstream.

bijujoseph commented on test/clinical-clusters/clinical-clusters-spec.js:
```diff
@@ -36,4 +36,74 @@ describe('benchmarks getter functions', () => {
+  it('claims 117 should not have any clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var clusters = data.filter(c => c.measureId === '117' && c.submissionMethod === 'claims');
+    assert.equal(0, clusters.length);
```
I see it in the docs but looks like the chai version we may be using is not compatible.
```
TypeError: assert.isEmpty is not a function
    at Context.it (test/clinical-clusters/clinical-clusters-spec.js:43:12)
```

kencheeto replied:
hmm - it's useful to have, want to try upgrading chai to 4.x?

bijujoseph commented:
@kencheeto, I addressed the comments. I got your point, over a period the tests will go crazy. I modified the tests to verify only certain cases in the generator. But, for that, I have to use some specific cases in the input file itself.

bijujoseph commented:
upgraded chai to 4.x, anyway later it will be useful. I have removed the over crowded tests, so `isEmtpy` is not being used by me.
